### ISSUE TITLE
Does not work for blocks with internal inventory

### ISSUE THREAD

**Issue by yatokiry** [Snownee/Jade#333]
### Mod loader

Forge

### Minecraft version

1.20.1

### Mod version

 Jade-1.20.1-forge-11.6.3.jar

### Modloader version

Forge - 47.2.4

### Modpack info

This Profile(Due to the sheer number of mods, I'm sending you a CurseForge profile.) : https://www.dropbox.com/scl/fi/qq2alw4zx75sf9xvzlhv2/MochiMochiRINYAServer-1.7.2.zip?rlkey=5lskr7ul72yarou96bzec938d&dl=0

### If bug:

- [ ] Can you reproduce this issue with relevant mods only?

### Issue description

I use automatic translation because I do not speak English.

Jade does not work well in the mod environment of the profile you sent.
Specifically, when using blocks with internal inventory or cables that can reference internal information, such as the ME Smart Cable in Applied Energistics 2, the Jade overlay does not appear, or does not switch to the block that was viewed before it. The situation is as follows.
I have tried changing the Jade config, deleting the config file, pressing the keypad (which may be pointless since it works on other blocks to begin with), installing an older version of Jade, etc., but no improvement.

**Comment by Thedrummonger** [comment]
I ran into this exact same issue in a modpack I'm putting together. Turns out the issue was cause by the Blue Power mod. I can replicate this issue with only the following
Minecraft 1.20.1
NeoForge 47.1.84
Jade 11.7.0
BluePower 0.10.323-alpha-universal

Unfortunately it looks like BluePower is shipped with it's own Hwyla compat and the mod hasn't been updated in six months, so I would hazard to guess this bug is on blue powers end. If you have blue power installed removing it will hopefully fix the issue.
